storage/sas_token.go

```diff
@@ -68,6 +69,27 @@ func ComputeAccountSASToken(accountName string,
 	return sasToken, nil
 }
 
+// ComputeAccountSASConnectionString computes the composed SAS Connection String for a Storage Account based on the
+// sas token
+func ComputeAccountSASConnectionString(accountName string, sasToken string) string {
+	return fmt.Sprintf(
+		"BlobEndpoint=https://%[1]s.blob.core.windows.net/;"+
+			"FileEndpoint=https://%[1]s.file.core.windows.net/;"+
+			"QueueEndpoint=https://%[1]s.queue.core.windows.net/;"+
+			"TableEndpoint=https://%[1]s.table.core.windows.net/;"+
+			"SharedAccessSignature=%[2]s", accountName, sasToken[1:]) // need to cut the first character '?' from the sas token
+}
+
+// ComputeAccountSASConnectionUrlForType computes the SAS Connection String for a Storage Account based on the
+// sas token and the storage type
+func ComputeAccountSASConnectionUrlForType(accountName string, sasToken string, storageType string) (string, error) {
```

Comment:
in general this library follows a pattern of returning a nil-object when there's an error, so that any attempt to access a bad result will panic - could we make this:

```suggestion
func ComputeAccountSASConnectionUrlForType(accountName string, sasToken string, storageType string) (*string, error) {
```

and then return `nil, fmt.Errorf..` below?